.circleci/config.yml

```diff
@@ -30,3 +31,11 @@ jobs:
                   name: Unit Tests
                   command: |
                       yarn test
+            - run:
+                  name: design pattern cypress tests
+                  command: |
+                       if [ $CIRCLE_BRANCH == $MASTER_BRANCH ] || [ $CIRCLE_BRANCH == $DEV_BRANCH ]; then
```

Comment:
This will only work if you define those environment variables for master branch and dev branch. I made a quick PR to convert this file to using workflows instead if you'd rather use that approach:
https://github.com/pxblue/react-design-patterns/pull/19